scripts/Tools/Makefile

```diff
@@ -908,6 +924,10 @@ install_lib: $(COMPLIB)
 	$(CP) -p $(COMPLIB) $(CSMSHARELIB)
 	$(CP) -p *.$(MOD_SUFFIX) *.h $(INCLUDE_DIR)
 
+install_lib_cpl7: $(COMPLIB)
+	$(CP) -p $(COMPLIB) $(CSMSHARELIB)
```

Comment:
Should CSMSHARELIB here be CSMSHARELIB_CPL7? Is this maybe the reason for the unintuitive line I called out above?

Reply:
Yes, using CSMSHARELIB_CPL7 here removes the need for the unintuitive line above.